draft-ietf-quic-tls.md

```diff
@@ -859,6 +865,11 @@ used for QUIC packet protection is AEAD that is negotiated for use with the TLS
 connection.  For example, if TLS is using the TLS_AES_128_GCM_SHA256, the
 AEAD_AES_128_GCM function is used.
 
+QUIC packets are protected prior to applying packet number encryption
+({{pn-encrypt}}).  Thus, the unprotected packet number is part of the AAD.  When
```

Comment:
s/AAD/AEAD/

Reply:
AAD is correct, but it's not clear and it's the first use of AAD, so I have reworded to use consistent terminology.